ones_and_zeros.js

```diff
@@ -0,0 +1,6 @@
+// https://www.codewars.com/kata/ones-and-zeros/train/javascript
+
+const binaryArrayToNumber = arr => {
+    const toBinary = arr.toString(2).replace(/[^0-9\.]+/g, "");
```

Comment:
1. The regexp in the `replace` could be simplified :smile: Think of what you want to replace (probably only the commas).
2. Is the argument for `toString` necessary?
3. What you are doing here is essentially what `Array.prototype.join` does :smile: Read up on it